16748.md Additions

Picklze1 opened the pull request:
Changes:
- Added things about 16748

Picklze1 committed:
Update 16748.md

olivia-banks commented:
Thanks for starting this! Would you please also include a few images before you finish? I trust you know how to insert images, but if not, add them to the assets folder under your team's subdirectory, and reference them like this:

```markdown
![Alt Text](/assets/16748/image.extension)
```

Picklze1 committed:
16748.md Quick edits

Changes
- Extended Outreach
- Expanded TODO for awards
- Added title for Projects

Picklze1 commented:
> Thanks for starting this! Would you please also include a few images before you finish? I trust you know how to insert images, but if not, add them to the assets folder under your team's subdirectory, and reference them like this:
> 
> ```
> ![Alt Text](/assets/16748/image.extension)
> ```

sounds good

Picklze1 committed:
Update index.md

IsaccBarker commented on content/index.md:
```diff
@@ -138,6 +138,8 @@ Spencer Brady
 
 Hasan Rahim
+
+Dylan Johnson De Lacy
```
Is this their(?) first and last name, or full name? I can't tell.

Picklze1 replied:
First and Last
